bin/crew

```diff
@@ -52,6 +57,8 @@ require_relative '../lib/package'
 require_relative '../lib/package_utils'
 require_relative '../lib/util'
 
+require_gem('highline/import')
+
 # Disallow sudo
 abort 'Chromebrew should not be run as root.'.lightred if Process.uid.zero?
```

Comment:
This should be above everything else and just below the require* lines too.  No need to do any processing if we need to exit early.